js/web/test/data/ops/expand.jsonc

```diff
@@ -0,0 +1,89 @@
+[
+  {
+    "name": "Expand 1 - float32",
+    "operator": "Expand",
+    "attributes": [],
+    "cases": [
+      {
+        "name": "Expand 1 - float32",
+        "inputs": [
+          {
+            "data": [1],
+            "dims": [1, 1],
+            "type": "float32"
+          },
+          {
+            "data": [1, 4],
+            "dims": [2],
+            "type": "int64"
+          }
+        ],
+        "outputs": [
+          {
+            "data": [1, 1, 1, 1],
+            "dims": [1, 4],
+            "type": "float32"
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "name": "Expand 2 - float32",
```

Comment:
Test for first bug fix - input tensors are of shape shapes but the second tensor's contents are different.